doc/classes/AnimatedSprite3D.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AnimatedSprite3D" inherits="SpriteBase3D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Sprite node in 3D that can use multiple 2D textures for animation.
 	</brief_description>
 	<description>
+		Sprite node in 3D that can use multiple 2D textures for animation. Animations are created using a [SpriteFrames] resource, which can be configured in the editor via the SpriteFrames panel.
```

Comment:
Avoid repeating the brief description: develop it or at least don't duplicate it. Avoid the passive voice. Example:
`Animations rely on a [SpriteFrames] resource and work the same as [AnimatedSprite]: use the SpriteFrames bottom panel in the editor to set them up..`